cdap-security/src/main/java/co/cask/cdap/security/securestore/FileSecureStoreMetadata.java

```diff
@@ -0,0 +1,188 @@
+/*
+ * Copyright © 2014 Cask Data, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+package co.cask.cdap.security.securestore;
+
+import co.cask.cdap.api.security.securestore.SecureStoreMetadata;
+import com.google.gson.stream.JsonReader;
+import com.google.gson.stream.JsonWriter;
+import org.apache.commons.io.Charsets;
+
+import java.io.ByteArrayInputStream;
+import java.io.ByteArrayOutputStream;
+import java.io.IOException;
+import java.io.InputStreamReader;
+import java.io.OutputStreamWriter;
+import java.util.Date;
+import java.util.HashMap;
+import java.util.Map;
+
+/**
+ * Represents the metadata for the data stored in the Secure Store.
+ */
+class FileSecureStoreMetadata implements SecureStoreMetadata {
```

Comment:
Same here, there is nothing file specific and I believe the `SecureStoreMetadata` can be just a POJO.